content/docs/alerting/configuration.md

```diff
@@ -64,10 +64,16 @@ global:
   # The default SMTP From header field.
   [ smtp_from: <tmpl_string> ]
   # The default SMTP smarthost used for sending emails.
+  # Provide the correct port for the smarthost:
+  # 25 for servers not providing TLS / SSL (set smtp_require_tls to "false" 
+  # 587 for STARTTLS servers
+  # 465 for SMTPS severs
+  # example: smtp.example.org:587
   [ smtp_smarthost: <string> ]
-  # SMTP authentication information.
+  # SMTP authentication information for SMTP Auth using PLAIN / LOGIN
```

Comment:
I don't think this is accurate, you need identity, username and password for gmail.